src/main/java/uk/ac/ebi/eva/pipeline/model/PopulationStatistics.java

```diff
@@ -15,17 +15,23 @@
  */
 package uk.ac.ebi.eva.pipeline.model;
 
+import org.springframework.context.annotation.Profile;
 import org.springframework.data.annotation.PersistenceConstructor;
 import org.springframework.data.mongodb.core.index.CompoundIndex;
 import org.springframework.data.mongodb.core.index.CompoundIndexes;
 import org.springframework.data.mongodb.core.mapping.Document;
 import org.springframework.data.mongodb.core.mapping.Field;
 
+import uk.ac.ebi.eva.pipeline.Application;
+
 import java.util.Map;
 
 /**
- * setters have package visibility, the user should use the constructor.
+ * Statistics related to a set of samples for a given variant.
+ * <p>
+ * Setters have package visibility, the user should use the constructor to instantiate valid objects.
  */
+@Profile(Application.MONGO_EXPERIMENTAL_PROFILE)
```

Comment:
The problematic class is the PopulationStatistics when it's automatically injected. There are no problems when the classes are manually injected, which is how it is done currently. 

In the future, when we actually use the PopulationStatistics class and related ones, we just have to remove the experimental profile.